docs/applications/configuration-management/use-salt-states-to-configure-a-lamp-stack-on-a-minion.md

```diff
@@ -180,7 +180,7 @@ Salt State Modules are used for settings across groups of Minions. To adjust a c
         salt '<hostname or Minion ID>' cmd.run "a2ensite example.com.conf"
         salt '<hostname or Minion ID>' cmd.run "service apache2 reload"
 
-<a href="/docs/applications/salt/salt-states-apache-mysql-php-fail2ban" target="_blank">
+<a href="/content/applications/salt/salt-states-apache-mysql-php-fail2ban" target="_blank">
```

Comment:
Check for HTML tags like this. Not sure why they're used.

Reply:
@cwlinode There may be cases where is is needed, but can you find all cases of `<a href=''>` tags and see if they can be replaced?

Reply:
@GuessWhoSamFoo All set: https://github.com/linode/docs/pull/1138